styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+    margin: 15;
+    font-family: Arial, Holvetica ;
+    background-image: linear-gradient(to bottom right,#D0F1F5, #D0F1F5,#A3DCE2, #1CD0E4,#52C1CD)
+  }
+  .topnav {
+    overflow: hidden;
+    background-color: #333;
+  }
+  .topnav a {
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+  .topnav a:hover {
+    background-color: #ddd;
+    color: black;
+  }
+  .active {
+    background-color: #4CAF50;
+    color: white;
+  }
+  .topnav .icon {
+    display: none;
+  }
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    right: 0;
+    text-align: left;
+  }
+  
+  @media screen and (max-width: 600px) {
+    .topnav.responsive {position: relative;}
+    .topnav.responsive .icon {
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+  }
+  
+  @media screen and (max-width: 600px) {
+    .topnav a:not(:first-child) { display: none;}
+    .topnav a.icon {
+      float: right;
+      display: block;
+    }
+  }
+  
+    #home img {
+     width: 100%;
+     object-fit: cover;
+     margin: 50px, 50px, 50px, 50px;
+     padding-top: 50px;
+     padding-bottom: 0;
+   }
+  #home {
+   display: block;
+  }
+  
+  h2 {
+   text-align: center;
+   font-size: 3rem;
+   color: #3B909A;
+   margin: 80px, 0, 20px, 0;
+  }
+  
+  #about {
+    background-color: white;
+    padding-top: 80px;
+    padding-bottom: 90px;
+    padding-left: 150px;
+    padding-right: 150px;
+    font-size: 20px;
+    font-family: Courier New;
+  }
+  
+  p {
+   font-size: 20px;
+   font-family: Courier New;
+  }
+  
+  #skills {
+    padding: 0;
+    display: flex;
+  }
+  
+  .left {
+    width: 50%;
+    float: left;
+    font-size: 70px;
+    background-color: #77C2CA;
+    text-align: center;
+  }
+  .right {
+    width: 50%;
+    float: right;
+    font-size: 70px;
+    background-color: white;
+    text-align: center;
+  }
+  
+  gallery {
```

Comment:
The projects are kind of squished next to each other in the mobile view. 
It would be better to display them vertically in a column in the mobile view 💡